django/chat/utils.py

```diff
@@ -477,6 +477,12 @@ def get_source_titles(sources):
     ]
 
 
+def batch(iterable, n=1):
+    l = len(iterable)
+    for ndx in range(0, l, n):
+        yield iterable[ndx : min(ndx + n, l)]
```

Comment:
Please don't use lowercase `l` as a variable name; I had to read this 10 times

Reply:
Hahaha, my bad. This was copied from StackExchange